scheduler/scripts/filter_demo.py

```diff
@@ -84,6 +91,7 @@
             print(f'--- Program {pid} is rejected.')
 
     resources = frozenset({
+            resource_service.lookup_resource('Gemini South'),
```

Comment:
This should probably `Site.GS` or something similar insted of `'Gemin South'`, but I'll go ahead and see if I can fix this in my PR.